docs/source/events.rst

```diff
@@ -449,3 +449,199 @@ events.
 - `Daemon tests <https://github.com/endlessm/eos-event-recorder-daemon/blob/efc6bb0e1283236ee4fe9c3d7fc992c4a53436d8/tests/daemon/test-daemon.c#L51>`_
 
   - ``350ac4ff-3026-4c25-9e7e-e8103b4fd5d8``
+
+
+Timestamp Algorithm
```

Comment:
Can this be a separate page? I don't think the timestamp algorithm is particularly related to the documentation of different events' payloads.

Reply:
OK.